modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -23,6 +23,7 @@ process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: meta.id
     def reference_cdna = fasta ? "--parse-cdnas $fasta" : ""
+    def VERSION = '1.1.1' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
```

Comment:
So the tool used to have a version string but no longer does?

Reply:
No, it was just harcoded differently :-)